lib/ui/process_ui/widgets/html_box_control.dart

```diff
@@ -172,11 +173,18 @@ class _HtmlRendererState extends State<HtmlRenderer> {
                 .langToCode(context.read<GlobalProvider>().chosenLang[i]));
       });
     }
+    String lang = context
+                .read<GlobalProvider>()
+                .langToCode(context.read<GlobalProvider>().chosenLang[context.watch<GlobalProvider>().htmlBoxTabIndex]);
+    log(lang);
     return SingleChildScrollView(
       child: Html(
         data:
             context.watch<GlobalProvider>().fieldDisplayValues[widget.field.id]
                 [context.watch<GlobalProvider>().htmlBoxTabIndex],
+        style: {
+     'html':  Style(textAlign: lang == 'ara'?TextAlign.right: TextAlign.left),
```

Comment:
there can be other left to right languages as well, so we need to handle based on that condition

Reply:
added the changes @jainhitesh9998